sudo-user.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+adduser --disabled-password --gecos '' user
+adduser user sudo
+echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
```

Comment:
Should this be moved into `RUN` statements in `Dockerfile`?
Should this (in whatever form) be moved into the/a base image, e.g., `pcic/geospatial-python`?